util/chplenv/chpl_hwloc_configs.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+import optparse, sys, os, re
+from sys import stdout
+
+import chpl_arch, chpl_compiler, chpl_platform
+import utils
+from utils import memoize
+
+@memoize
+def get_uniq_cfg_path():
+    target_platform = chpl_platform.get('target')
+    target_compiler = chpl_compiler.get('target')
+    target_arch = chpl_arch.get('target', map_to_compiler=True, get_lcd=False)
+    uniq_cfg_path = '{0}-{1}-{2}'.format(target_platform, target_compiler,
```

Comment:
Yeah, I could go for that. It looks like only qthreads and gasnet (which I haven't done yet) differ from that. I'll make this change.